src/vit-testing/vitup/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2021"
 [dependencies]
 slave-pool = "0.2.3"
 ipfs-api = "0.16.0"
+function_name = "0.3.0"
```

Comment:
think this is unneeded (I tried `cargo test --no-run` and it succeeded without it)